content/docs/user-guide/project-structure/configuration.md

```diff
@@ -111,6 +111,11 @@ within:
   <abbr>DVC files</abbr> created or modified by DVC commands. The files will not
   be committed. Accepts values `true` and `false` (default).
 
+- `core.site_cache_dir` - override
+  [platformdirs.site_cache_dir](https://platformdirs.readthedocs.io/en/latest/api.html#platformdirs.site_cache_dir)
```

Comment:
Tbh it's not exactly clear what does it do. Does it mean for example that it affects other place where site dir is used besides our index files?

Reply:
> Does it mean for example that it affects other place where site dir is used besides our index files?

Like outside of dvc context?

Reply:
Like even in DVC context. It sounds a bit like we are monkey patching the lib or in some other way make it return a different value (is it true?) . I guess my question here - do we even need to refer to the lib (it's an implementation details after all). Users need to know from the product perspective what it affects, when it can be used.